### ISSUE TITLE
Request: Minimum file size

### ISSUE THREAD

**Issue by neilicious** [DobyTang/LazyLibrarian#661]
In addition to the maximum file size option that currently exists, would it be possible to add a minimum file size option?

I'm currently using LL to auto-download audiobooks and for the most part I'm able to do this just by reversing the default options (ie. rejecting epub, mobi etc., whilst accepting mp3 and m4b). Unfortunately, on occasion, someone will upload an ebook torrent which includes multiple book formats. Typically they then don't have any format in the title of the torrent and LL will snatch that as it has no reason to reject it. If I could demand a minimum file size for the torrent I could ensure I wasn't getting ebooks by accident.

**Comment by philborman** [comment]
Haven't tried using LL for audiobooks, but minimum file size is easy to
fake if you don't mind editing a bit.
lazylibrarian/searchnzb.py line 170
lazylibrarian/searchtorrents.py line 173
lazylibrarian/searchrss.py line 143
change the > to <
so it treats the configured value as a minimum instead of a maximum. If
this works ok I will add a minimum option.

**Comment by neilicious** [comment]
Tested with the changes that you suggested and that works perfectly. Tested it on the exact situation outlined in my initial comment and it correctly selected the audiobook and ignored the multi-ebook torrent. Then tested it on a book where the audiobook isn't yet available but the ebooks are (again in a multi-ebook torrent) and it rightly didn't snatch anything, just left the book as wanted still.

**Comment by philborman** [comment]
Ok. Will add that in the next release. Probably tomorrow.

**Comment by philborman** [comment]
Added that feature now.
If you are in a git install you will need to revert the manual changes you made to the three search***.py files if you want to use the internal upgrade system, git will complain you have local changes otherwise. If you installed from zip you should be ok.

**Comment by bcsteeve** [comment]
Is it possible to remove the requirement that the minimum value be an integer?  Alternatively, change it from Mb to Kb?  I only just installed LL and on my very first test, I was getting a lot of failures in SAB with the NZB being empty.  For whatever reason, the indexers I'm using have (at least some) 0.00Mb sized (empty) files.  LL grabs that one and it fails and never tries again it seems.  I changed Min size to 1 and that solved it... but the problem there is that it is quite conceivable (common even) for ebooks to be less than 1 Mb in size.
